src/olympia/amo/cron.py

```diff
@@ -57,6 +58,12 @@ def days_ago(days):
                 pass
         file_upload.delete()
 
+    stale_scanner_results = ScannerResult.objects.filter(
+        upload=None, version=None
+    ).order_by('id')
+    for scanner_result in stale_scanner_results:
+        scanner_result.delete()
+
```

Comment:
unless there's custom logic in the model's `delete` method, you can just do `ScannerResult.objects.filter(upload=None, version=None).delete()`

Reply:
Oh, I did not know that. Thanks!